### DISCUSSION TITLE
Student ID getting Rejected | Vidyalankar Institute of International Education

### DISCUSSION THREAD

**Discussion by Abood2284** [community/community#69075] [GitHub Education]
### Hi everyone,

Hi everyone, my college has just begun, and I have received a student email, and a student ID card.
But my student ID card is getting rejected, for some reason saying the date is not written on it. I don;t know why the date isn't mentioned and could be done.

![IMG_8821](https://github.com/community/community/assets/90205565/bb2a5276-26bb-4984-8f3a-903af0a1a709)

**Comment by vishalkrishna8** [comment]
Reach out to your college's administrative office, preferably the one responsible for issuing student ID cards. Inquire about the absence of a date on your card and ask whether it can be added or if there's a specific reason it's omitted.
